include/links.h

```diff
@@ -0,0 +1,61 @@
+#ifndef LINKS_H_
+#define LINKS_H_
+#include <vector>
+#include <utility>
+#include <string>
+namespace newsboat {
+// This enum has to be kept in sync with enum LinkType in rust/libnewsboat/src/htmlrenderer.rs
```

Comment:
Please also move rust/libnewsboat/src/htmlrenderer.rs to links.rs, update a comment in it to point to links.h, and update rust/libnewsboat/src/mod.rs to replace `htmlrenderer` with `links`.